backend/substrapp/tasks/k8s_backend.py

```diff
@@ -273,9 +279,12 @@ def _k8s_build_image(path, tag, rm, cache_index):
     args = [
         f'--dockerfile={dockerfile_fullpath}',
         f'--context=dir://{path}',
-        f'--destination={REGISTRY}/{tag}:substra',
+        f'--destination={REGISTRY}/substra/substra:{tag}',
         f'--cache={str(not(rm)).lower()}',
         '--snapshotMode=redo',
+        '--use-new-run',
+        '--push-retry=3',
+        '--cache-copy-layers',
```

Comment:
I have not measured the real speed as we need big compute plan to see if it's better. But e2e substra-tests took the same amount of time :)

Reply:
You could try testing with substra-tools instead of substra-tools-minimal ( cache warmup, image prepopulate, and substra-tests in factory.py)

Reply:
Using substra-tools instead of minimal with `pytest -rs -v --durations=0 tests/test_execution.py -n 5`

`improve-worker`
0:15:32
0:14:17
0:13:33
0:15:06

`master`
0:17:35
0:19:07
0:18:01
0:17:11

Reply:
Great results!